platforms/software/build-init/src/main/java/org/gradle/buildinit/plugins/internal/ProjectLayoutSetupRegistry.java

```diff
@@ -100,7 +100,7 @@ public BuildGenerator getDefault() {
     public BuildInitializer get(String type) {
         if (!registeredProjectDescriptors.containsKey(type)) {
             TreeFormatter formatter = new TreeFormatter();
-            formatter.node("The requested build type '" + type + "' is not supported. Supported types");
+            formatter.node("The requested build type: '" + type + "' is not supported. Supported types");
```

Comment:
I think this was OK without the colon because the sentence is "The requested build type is not supported".

Maybe we need a colon at the end of "Supported types"?

Reply:
The colon at the end is automatically added by the formatter.  I'll remove the earlier colon.